src/Makefile.am

```diff
@@ -92,6 +92,7 @@ libavrdude_a_SOURCES = \
 	avr.c \
 	avr910.c \
 	avr910.h \
+        avrcache.c \
```

Comment:
A tiny formatting error here. Too many spaces used as indent.

Reply:
:) The old tab/space mixup. I had not anticipated this list be written with tabs.